addons/sale/migrations/12.0.1.1/pre-migration.py

```diff
@@ -3,6 +3,12 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openupgradelib import openupgrade
 
+_column_copies = {
+    'sale_order_line': [
+        ('qty_delivered', 'qty_delivered_manual', None),
```

Comment:
This should be put as 0 for all the non manual methods. If not, there can be side effects if this is the expected value.

Reply:
The qty_delivered_manual for non manual methods will be 0 due to `_inverse_qty_delivered` method.

Reply:
Yes, that's my point, so copying the column is not the solution.